src/api/public/apidocs-new/paths/source_project_name_package_name.yaml

```diff
@@ -1,3 +1,352 @@
+get:
+  summary: List source files of a package.
+  description: Get a list of source files belonging to a package.
+  security:
+    - basic_authentication: []
+  parameters:
+    - $ref: '../components/parameters/project_name.yaml'
+    - $ref: '../components/parameters/package_name.yaml'
+    - in: query
+      name: deleted
+      schema:
+        type: string
+      description: |
+        Set to `1` to list source files of deleted packages.
+
+        Return a "Bad Request" error (400) if it is set to `1` and the package exists.
+      example: 1
+    # TODO
+    # - in: query
+    #   name: emptylink
+    #   schema:
+    #     type: string
+    #     enum:
+    #       - 1
+    #       - 0
+    #   default: 0
+    #   description:
+    #   example: 1
```

Comment:
I will leave documenting the `emptylink` parameter as a TODO until I figure out what is it for.